doc/classes/TileSetAtlasSource.xml

```diff
@@ -80,6 +80,13 @@
 				Returns how many animation frames has the tile at coordinates [param atlas_coords].
 			</description>
 		</method>
+		<method name="get_tile_animation_sync" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="atlas_coords" type="Vector2i" />
+			<description>
+				Returns if each animation start at first, or random frame.
```

Comment:
```suggestion
				Returns [code]true[/code] if animations are synchronized across all tiles for the tile at [param atlas_coords], [code]false[/code] otherwise. See also [method set_tile_animation_sync].
```